rhc-worker-playbook.spec

```diff
@@ -22,6 +22,7 @@ BuildRequires: python3-devel
 BuildRequires: python3dist(pip)
 BuildRequires: python3dist(wheel)
 BuildRequires: python3dist(setuptools)
+BuildRequires: python3dist(pyyaml)
```

Comment:
hm is pyyaml needed as system package, if build it as wheel from the vendored sources?

Reply:
Good point. This must be left over from previous iterations when I was trying to use the system PyYAML. It builds just fine without this requirement.